src/techui_builder/generate.py

```diff
@@ -59,16 +60,80 @@ def _get_screen_dimensions(self, file: str) -> tuple[int, int]:
 
         return (height, width)
 
+    def _get_screen_dimensions_object(
```

Comment:
This should probably be called something like get widget dimenstions to avoid confusion
```suggestion
    def _get_widget_dimensions(
```

Reply:
now _get_widget_screen_dimensions, and _get_file_screen_dimensions, to show they operate mainly the same but for files and widget objects